compose.yml

```diff
@@ -17,16 +17,16 @@ services:
       - TS_TUNE_MAX_CONNS=500
     volumes:
       # needed for normal image
-      #- ./assume-db:/var/lib/postgresql/data
+      - ./assume-db:/var/lib/postgresql/data
```

Comment:
why was the docker file changed?

Reply:
Oh sorry, for my laptop the other image never works, I commited it accidently